.github/workflows/crowdin.yml

```diff
@@ -20,22 +20,29 @@ jobs:
     if: github.event_name == 'schedule'
     uses: nuxeo/nuxeo-coldstorage/.github/workflows/crowdin.yml@lts-2023
     secrets: inherit
+    needs: crowdin
```

Comment:
Is it to run after the `crowdin` job?

Reply:
Yes, to avoid conflict (409) regarding the Crowdin translation file (same fior all GitHub branches).